10.2/Dockerfile.fedora

```diff
@@ -28,7 +28,13 @@ LABEL summary="MariaDB is a multi-user, multi-threaded SQL database server" \
       version="$VERSION" \
       release="$RELEASE.$DISTTAG" \
       architecture="$ARCH" \
-      maintainer="SoftwareCollections.org <sclorg@redhat.com>"
+      maintainer="SoftwareCollections.org <sclorg@redhat.com>" \
+      usage="docker run -e MYSQL_USER=<user_name> -e MYSQL_PASSWORD=<password> -e MYSQL_DATABASE=<db_name> -e MYSQL_ROOT_PASSWORD=<root_password> -p 3306:3306 mariadb" \
```

Comment:
IMHO there is no image named "mariadb". I think `$FGC/$NAME` should be used.

Reply:
Very good point!